packages/router/test/int/PendlePtToAsset.eth.spec.ts

```
@@ -0,0 +1,541 @@
+    ptToken,
+    assetToken,
+    syToken,
+    router,
+    adapter,
+    owner,
+    user,
+  };
+}
+
+// Tests for running in ethereum mainnet fork
+describe.only('PendlePtToAssetAdapter', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)